levels.go

```diff
@@ -1439,6 +1439,18 @@ func (s *levelsController) runCompactDef(id, l int, cd compactDef) (err error) {
 		return err
 	}
 
+	getSizes := func(tables []*table.Table) int64 {
+		size := int64(0)
+		for _, i := range tables {
+			size += i.Size()
+		}
+		return size
+	}
+
+	sizeNewTables := getSizes(newTables)
+	sizeOldTables := getSizes(cd.bot) + getSizes(cd.top)
+	y.NumBytesCompactionWrittenAdd(s.kv.opt.MetricsEnabled, sizeNewTables)
```

Comment:
this whole section should be in a conditional of if Log.Info or MetricsEnabled ?

Reply:
in my experience later never comes, so i suggest to change now than kicking the can down the road